sub-9605/anat/sub-9605_run-2_T2starw.json

```diff
@@ -3,5 +3,5 @@
 	"MagneticFieldStrength": 7,
 	"Manufacturer": "Siemens",
 	"InstitutionName": "Mount Sinai",
-	"AcquisitionDuration": "167"
-}
\ No newline at end of file
+	"AcquisitionDuration": "507"
```

Comment:
why 507?
i calculated from Alan's message: 60*8 + 21=501

Reply:
oups I saw 27s. I will fix it